.github/workflows/format.yml

```diff
@@ -0,0 +1,31 @@
+name: Format
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    working-directory: javascript
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14.x"
+      - name: Format
+        run: |
+          npm ci
+          npm run prettier
```

Comment:
Re-upping this: I think it will be confusing to have a CI task pushing formatting changes to a PR.  The author will then have to merge from GitHub before adding further changes to the PR and it could lead to weird conflicts.  I think this should just be a check that fails if the code is not formatted.  If that is not possible, probably the CI job should just be removed for now (though then the guarantees about having formatted code are lost).